src/Repository/ProjectRepository.php

```diff
@@ -59,12 +59,23 @@ public function addConditionState(QueryBuilder $query, string $state): void
     }
 
     /**
-     * Filter projects according to a Shuttle.
+     * Filter projects according to a shuttle.
      */
     public function addConditionShuttle(QueryBuilder $query, Shuttle $shuttle): void
     {
         $query
             ->andWhere('p.shuttle = :shuttle')
             ->setParameter('shuttle', $shuttle->getId());
     }
+
+    /**
+     * Filter projects according to members.
+     */
```

Comment:
pass array of Team (add docblock=



```suggestion
    /**
     * Filter projects according to members.
     *
     * @var Collection $members
     */
```